### ISSUE TITLE
FATAL ERROR: 'v8::Isolate::LowMemoryNotification' unimplemented

### ISSUE THREAD

**Issue by alexlamsl** [nodejs/node-chakracore#433]
* **Version**: v8.6.0
* **Platform**: Windows 10 x64

Trying to process a bunch of file I/O in parallel and got:
```
FATAL ERROR: 'v8::Isolate::LowMemoryNotification' unimplemented
```

**Comment by digitalinfinity** [comment]
Thanks @alexlamsl - indeed, this is a gap in our shim that we are aware of. Can you share your test case that hit this issue- I want to get a sense of how common this could be to help prioritize it.

**Comment by obastemur** [comment]
We better make https://github.com/nodejs/node-chakracore/blob/8cf4aa996118b7bc17efc2f4fdb170186d1ed3bf/deps/chakrashim/src/jsrtutils.cc#L909 DEBUG only?

**Comment by addaleax** [comment]
When I introduced the call to `LowMemoryNotification()` in the first place, that was more of a nice-to-have thing rather than a necessity. If there’s no simple way to do this in the shim, you can just make it a no-op.

**Comment by obastemur** [comment]
Agree. `abort` is too much on production.

**Comment by addaleax** [comment]
To be clear, I meant making `LowMemoryNotification()` a no-op. Crashing when encountering unimplemented functions seems perfectly fine to me in general.

**Comment by alexlamsl** [comment]
@digitalinfinity getting a reproducible test case is quite a challenge, as the code only crashes on one of dozen of machines I run the same program on.
